src/animation.rs

```diff
@@ -435,9 +435,9 @@ fn update_animation_state(
 
     let range = maybe_tag
         .map(|t| *t.range.start()..*t.range.end())
-        .unwrap_or(0..aseprite.frame_durations.len() as u16);
+        .unwrap_or(0..(aseprite.frame_durations.len() - 1) as u16);
```

Comment:
I needed to subtract this by 1. Feel free to suggest alternative approaches. Here is the context:

When using animations without specifying a tag it would never submit the `AnimationFinished` event. E.g. If we play the entire animation of an explosion (where there is no need for tags since we are using a single aseprite for the explosion), it would get stuck on the last frame.

If we specified a tag, it worked as expected. Turned out that with tag and without tag produced two different ranges.

This is because tags are loaded via `InclusiveRange` while the rest of the app uses `Range`. By treating the two ranges (with tag and without) as the same, we can match our ranges.

Reply:
I might suggest a proper fix here is to unite/standardise the range behaviour